### ISSUE TITLE
MQTT JSON formatting issue

### ISSUE THREAD

**Issue by rroblik** [fredlcore/bsb_lan#262]
Dear @fredlcore,

I'm trying to use MQTT with bsb_lan but I have seems to have a JSON formatting issue (`byte mqtt_mode = 3;`) :

From the serial monitor for example : 

 ```
23:35:49.729 -> Output topic: BSB-LAN/json
23:35:49.729 -> Payload Output : {"PAC":{"id":8700","name":"STR8700_TEXT","value": "","desc": "","unit": "UNIT_DEG_TEXT","error": 261}}
23:35:52.728 -> Output topic: BSB-LAN/json
23:35:52.728 -> Payload Output : {"PAC":{"id":8743","name":"STR8743_TEXT","value": "","desc": "","unit": "UNIT_DEG_TEXT","error": 261}}
```

The `"` after `8700` break json format so my MQTT broker client see it as string instead of parseable JSON.

From what I see the error is here for all `id` values.

Note : i'm not connected to the bsb bus for now, that why I've 261 errors.

Can you help ?

**Comment by fredlcore** [comment]
That's an error, yes. @dukess: Do you know how to fix it? Otherwise I'll look into it.

**Comment by dukess** [comment]
Done. https://github.com/fredlcore/bsb_lan/pull/263

Sleepless night.

**Comment by rroblik** [comment]
Impressive !

thanks a lot @dukess for the hard work. 

Waiting the merge 👌🏽

**Comment by dukess** [comment]
Thanks for report!

**Comment by rroblik** [comment]
Can be closed #263 

Thanks !
